examples/features/load_balancing/client/main.go

```diff
@@ -84,11 +89,24 @@ func main() {
 
 	fmt.Println("--- calling helloworld.Greeter/SayHello with round_robin ---")
 	makeRPCs(roundrobinConn, 10)
+	// You can also plug in your own custom lb policy, which needs to be
+	// configurable. This n is configurable. Try changing it and see how the
+	// behavior changes.
+	customroundrobinConn, err := grpc.Dial(
+		fmt.Sprintf("%s:///%s", exampleScheme, exampleServiceName),
+		grpc.WithDefaultServiceConfig(`{"loadBalancingConfig": [{"custom_round_robin":{"n": 3}}]}`),
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
+	)
+	if err != nil {
+		log.Fatalf("did not connect: %v", err)
+	}
+	defer customroundrobinConn.Close()
+	fmt.Println("--- calling helloworld.Greeter/SayHello with custom_round_robin ---")
+	makeRPCs(customroundrobinConn, 20)
 }
 
 // Following is an example name resolver implementation. Read the name
 // resolution example to learn more about it.
-
 type exampleResolverBuilder struct{}
```

Comment:
I don't believe this example intends to show off custom resolvers - can this be replaced with the manual resolver instead?  We may not have had it when this was created.

Reply:
Switched.